pkgs/games/anki/default.nix

```diff
@@ -52,8 +53,6 @@ let
     buildAndTestSubdir = "build/runner";
 
     nativeBuildInputs = [ pkg-config ];
-
-    doCheck = false;
```

Comment:
Why?

Reply:
It was erroring out originally, and I forgot to get back and fix it before marking the pr ready, sorry!

I was fixing that up locally already, and just pushed a second commit which fixes all the checks, apologies for not doing that before asking for review